src/Topics/StandardModernPrecision/Bids1C.hs

```diff
@@ -107,6 +133,13 @@ b1C1N = do
     makeCall $ T.Bid 1 T.Notrump
 
 
+b1C1Nalt :: Action  -- Alternative choice: swap the meanings of b1C1N and b1C2H
+b1C1Nalt = do
+    _slamInterest
+    minSuitLength T.Hearts 5
```

Comment:
Suppose you're 5-5 in two suits? Which one do you bid? Figure it out, and modify all these `Action`s. but that can be a separate PR.